polaris-react/src/components/ActionList/ActionList.module.css

```diff
@@ -1,8 +1,60 @@
+/* stylelint-disable selector-max-class, selector-max-specificity, selector-max-combinators, selector-max-compound-selectors */
+
+@media (--p-breakpoints-md-down) {
```

Comment:
I'd love to avoid using a non-mobile first media query here but it would require almost double the amount of code to do so... 🤔